keeper/src/main/scala/zio/keeper/swim/protocols/FailureDetection.scala

```diff
@@ -3,22 +3,33 @@ package zio.keeper.swim.protocols
 import upickle.default._
 import zio.clock.Clock
 import zio.duration._
-import zio.keeper.{ ByteCodec, NodeAddress }
 import zio.keeper.swim.Nodes.{ NodeState, _ }
-import zio.keeper.swim.{ ConversationId, Message, Nodes, Protocol }
+import zio.keeper.swim.{
+  ConversationId,
+  LocalHealthMultiplier,
+  Message,
+  MessageAcknowledge,
+  Nodes,
+  Protocol,
+  SuspicionTimeout
+}
+import zio.keeper.{ ByteCodec, NodeAddress, swim }
 import zio.logging._
-import zio.stm.TMap
+import zio.stm.{ STM, TMap }
 import zio.stream.ZStream
 import zio.{ Schedule, ZIO, keeper }
 
 sealed trait FailureDetection
 
 object FailureDetection {
 
-  final case object Ping                        extends FailureDetection
-  final case object Ack                         extends FailureDetection
-  final case object Nack                        extends FailureDetection
-  final case class PingReq(target: NodeAddress) extends FailureDetection
+  final case object Ping                                           extends FailureDetection
+  final case object Ack                                            extends FailureDetection
+  final case object Nack                                           extends FailureDetection
```

Comment:
Drop final in front of these to avoid future compat issues.

Reply:
done